integration/spark2/src/main/scala/org/apache/spark/sql/CarbonEnv.scala

```diff
@@ -54,6 +54,7 @@ class CarbonEnv {
 
   def init(sparkSession: SparkSession): Unit = {
     sparkSession.udf.register("getTupleId", () => "")
+    sparkSession.udf.register("getPositionId", () => "")
```

Comment:
it is not a good idea to add here, better add in CarbonEnvPreListner to register